run-be-ut.sh

```diff
@@ -117,8 +117,8 @@ if [[ -z ${USE_AVX2} ]]; then
     USE_AVX2=ON
 fi
 if [[ -z ${USE_AVX512} ]]; then
-    # Disable it by default
-    USE_AVX512=OFF
+    # Enable it by default
```

Comment:
sorry, why change this ? I'm afraid it could not be enabled by default, bc many users don't deploy SR with AVX512

Reply:
> sorry, why change this ? I'm afraid it could not be enabled by default, bc many users don't deploy SR with AVX512
I change it for checking the new implementation in the CI. will remove it soon